web/styles/typing_notifications.css

```diff
@@ -9,3 +9,10 @@
     list-style: none;
     margin: 0;
 }
+
+#mark_as_read_turned_off_banner {
+    /*  override the margin of main-view-banner since it causes
+    more gap than we want between mark as read banner and typing
+    notification.  */
```

Comment:
This is not our standard whitespace pattern for comments in CSS.

Reply:
oops, updated